client/app/main/main.js

```diff
@@ -10,3 +10,21 @@ angular.module('observatory3App')
         controllerAs: 'main'
       });
   });
+
+function getEmail() {
+	var coded = "rTTPmCfUdTPB@PrTB.CT"
+    var key = "gZ8fYCPqeJjtpaFI1GBLNyuoxA7XclKd4UWSkzQ63HMThnVirbvm5R2EOD9ws0"
```

Comment:
We set the environment variables in https://github.com/rcos/Observatory3/blob/master/server/config/local.env.sample.js (Copy the tracked local.env.sample.js to local.env.js for any private keys & emails, but make sure to include dummy values in the sample file)

Then expose the environment variables in https://github.com/rcos/Observatory3/blob/master/server/config/environment/index.js